dash.el

```diff
@@ -1894,6 +1938,36 @@ plist-like key-value pairs, similarly to &keys keyword of
 This binds N values from the list to a1 ... aN, then interprets
 the cdr as a plist (see key/value matching above).
 
+A shorthand notation for kv-destructuring exists which allows the
+patterns be optionally left out and derived from the key name in
+the following fashion:
+
+- a key :foo is converted into `foo' pattern,
+- a key 'bar is converted into `bar' pattern,
+- a key \"baz\" is converted into `baz' pattern.
+
+That is, the entire value under the key is bound to the derived
```

Comment:
IMHO "under the key" is a bit awkward.  Perhaps this would be clearer:

> That is, the entire value mapped to by the key...

Reply:
Maybe you could do a PR :D Feel free to improve the docs, they sure are sloppy at times.  Don't forget to run `./create-docs.sh`!

Reply:
> Feel free to improve the docs, they sure are sloppy at times. 

No promises, but I'll keep that in mind.

Reply:
FWIW, I've never heard of a value existing under a key, and I wouldn't say "the entire value mapped to by the key", as this is slightly ambiguous (keys don't actively map to values, rather they are mapped to values by the containing data structure).

I think for Elisp it's more natural to say things like "the value associated with the key", "the value corresponding to the key", etc.

(Disclaimer: I'm not a native English speaker :)